src/flick_auth/views.py

```diff
@@ -86,6 +89,26 @@ def get(self, request, pk):
         return success_response(self.serializer_class(profile, context={"request": self.request}).data)
 
 
+class UserLikedLstsView(generics.GenericAPIView):
+    model = Profile
+    serializer_class = MeLstSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def get(self, request, pk):
+        profile = Profile.objects.filter(user__id=pk).prefetch_related(
+            Prefetch(
+                "likes",
+                queryset=Like.objects.filter(like_type="list_like").prefetch_related(
+                    "lst", "lst__owner", "lst__collaborators", "lst__shows"
+                ),
+                to_attr="lst_likes",
+            )
+        )[0]
```

Comment:
oh shit this looks intense, :o is this the right way to optimize django stuff? like i thought it was just `prefetch_related("fields here")`